src/services/createRouter.spec.ts

```diff
@@ -462,3 +441,171 @@ test('initial route is not set until the router is started', async () => {
 
   expect(router.route.name).toBe('root')
 })
+
+describe('router.resolve', () => {
```

Comment:
this tests come from the now defunct `createRouterFind` and `createRouteResolve` spec files